includes/data-stores/class-wc-admin-reports-products-data-store.php

```diff
@@ -287,4 +296,93 @@ protected function get_cache_key( $params ) {
 		return 'woocommerce_' . self::TABLE_NAME . '_' . md5( wp_json_encode( $params ) );
 	}
 
+	/**
+	 * Make an entry in the wc_admin_order_product_lookup table for an order.
```

Comment:
Same comment, uhhhh, on the comment saying Make/Update. Also thoughts on unit tests here?

Reply:
Similar comment to above.  Unit tests here will need order status examples as well as refund data.